examples/decoding/plot_haxby_space_net.py

```diff
@@ -22,17 +22,17 @@
 condition_mask_train = np.logical_and(condition_mask, labels['chunks'] <= 6)
 condition_mask_test = np.logical_and(condition_mask, labels['chunks'] > 6)
 
-### make X (design matrix) and y (response variable)
+# make X (design matrix) and y (response variable)
 import nibabel
 from nilearn.image import index_img
-niimgs  = nibabel.load(data_files.func[0])
+niimgs = nibabel.load(data_files.func[0])
 X_train = index_img(niimgs, condition_mask_train)
 X_test = index_img(niimgs, condition_mask_test)
 y_train = target[condition_mask_train]
 y_test = target[condition_mask_test]
 
 
-### Loop over Graph-Net and TV-L1 penalties ####################################
+### Loop over Graph-Net and TV-L1 penalties ###################################
```

Comment:
> Yeah I don't really understand what the rule is ...

I hate autopep8 and other tools like this, because their interpretation
of the details of pep8 varies (and varies across versions of the same
tool).